tools/src/main/scala/org/apache/spark/tools/GenerateMIMAIgnore.scala

```diff
@@ -133,14 +140,14 @@ object GenerateMIMAIgnore {
     val (privateClasses, privateMembers) = privateWithin("org.apache.spark")
     val previousContents = Try(File(".generated-mima-class-excludes").lines()).
       getOrElse(Iterator.empty).mkString("\n")
-    File(".generated-mima-class-excludes")
-      .writeAll(previousContents + privateClasses.mkString("\n"))
+    File(".generated-mima-class-excludes").writeAll(
```

Comment:
Here, I decided to sort things just to make debugging a little nicer for myself.